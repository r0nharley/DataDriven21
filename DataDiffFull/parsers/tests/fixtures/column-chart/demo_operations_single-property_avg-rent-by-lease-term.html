<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="829" height="383">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-w9nkk1v-416">
            <rect x="0" y="0" width="753" height="299" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-w9nkk1v-417">
            <rect x="0" y="0" width="753" height="299" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="829" height="383" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="66" y="10" width="753" height="299"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 216.5 10 L 216.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 366.5 10 L 366.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 517.5 10 L 517.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 667.5 10 L 667.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 818.5 10 L 818.5 309" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 65.5 10 L 65.5 309" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 309.5 L 819 309.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 249.5 L 819 249.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 189.5 L 819 189.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 130.5 L 819 130.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 70.5 L 819 70.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 9.5 L 819 9.5" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="66" y="10" width="753" height="299"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 216.5 309 L 216.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 366.5 309 L 366.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 517.5 309 L 517.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 667.5 309 L 667.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 818.5 309 L 818.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 65.5 309 L 65.5 319" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 66 309.5 L 819 309.5"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 66 10 L 66 309"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-column-series highcharts-color-undefined highcharts-tracker" transform="translate(66,10) scale(1 1)" width="299" height="753" clip-path="url(https://staging.rentlytics.com/demo/bi/operations-single-property/visualization#highcharts-w9nkk1v-417)">
            <rect x="38.5" y="84.5" width="73" height="5" fill="#9b9bd7" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="189.5" y="239.5" width="73" height="6" fill="#9b9bd7" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="339.5" y="72.5" width="73" height="2" fill="#9b9bd7" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="490.5" y="66.5" width="73" height="10" fill="#9b9bd7" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="641.5" y="78.5" width="73" height="3" fill="#9b9bd7" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="299" height="753" clip-path="none"></g>
        <g class="highcharts-series highcharts-series-1 highcharts-column-series highcharts-color-undefined highcharts-tracker " transform="translate(66,10) scale(1 1)" width="299" height="753" clip-path="url(https://staging.rentlytics.com/demo/bi/operations-single-property/visualization#highcharts-w9nkk1v-417)">
            <rect x="38.5" y="89.5" width="73" height="150" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="189.5" y="71.5" width="73" height="168" fill="#218a8c" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="339.5" y="74.5" width="73" height="165" fill="#218a8c" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="490.5" y="76.5" width="73" height="163" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="641.5" y="81.5" width="73" height="158" fill="#218a8c" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-1 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="299" height="753" clip-path="none"></g>
    </g>
    <g class="highcharts-legend" transform="translate(284,342)">
        <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="261" height="26" visibility="visible"></rect>
        <g>
            <g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-0" transform="translate(8,3)">
                    <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">
                        <tspan>Loss to Lease</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#9b9bd7" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-column-series highcharts-color-undefined highcharts-series-1" transform="translate(132.953125,3)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Avg Actual Rent</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#218a8c" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
            </g>
        </g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="141.3" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:152px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="328" opacity="1">
            <tspan><3</tspan>
        </text>
        <text x="291.9" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:152px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="328" opacity="1">
            <tspan>03-06</tspan>
        </text>
        <text x="442.49999999999994" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:152px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="328" opacity="1">
            <tspan>06-09</tspan>
        </text>
        <text x="593.1" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:152px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="328" opacity="1">
            <tspan>09-12</tspan>
        </text>
        <text x="743.6999999999999" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:152px;text-overflow:clip;" text-anchor="middle" transform="translate(0,0)" y="328" opacity="1">
            <tspan>12+</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="314" opacity="1">
            <tspan>$-500</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="254" opacity="1">
            <tspan>$0</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="194" opacity="1">
            <tspan>$500</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="134" opacity="1">
            <tspan>$1,000</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="74" opacity="1">
            <tspan>$1,500</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:264px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="15" opacity="1">
            <tspan>$2,000</tspan>
        </text>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(376,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 61 0 67 -6 73 0 135 0 C 135 0 135 0 135 0 L 135 102 C 135 102 135 102 135 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>