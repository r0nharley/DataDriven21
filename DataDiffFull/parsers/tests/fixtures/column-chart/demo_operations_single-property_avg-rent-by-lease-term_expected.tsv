<3	Loss to Lease	41.8060200669	#9b9bd7
<3	Avg Actual Rent	1254.18060201	#218a8c
03-06	Loss to Lease	-50.1672240803	#9b9bd7
03-06	Avg Actual Rent	1404.68227425	#218a8c
06-09	Loss to Lease	16.7224080268	#9b9bd7
06-09	Avg Actual Rent	1379.59866221	#218a8c
09-12	Loss to Lease	83.6120401338	#9b9bd7
09-12	Avg Actual Rent	1362.87625418	#218a8c
12+	Loss to Lease	25.0836120401	#9b9bd7
12+	Avg Actual Rent	1321.07023411	#218a8c